independent-publisher-connectors/NFTMania/MintNFT.swagger.json

```diff
@@ -0,0 +1,84 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MintNFT",
+    "description": "NFT Mania provides you with an Ultimate NFT experience and lets you develop applications in no time!",
+    "version": "1.0"
+  },
```

Comment:
Hi @shreyan1999, Please add a contact section, please see our templates [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json)

Reply:
Sorry for not providing it. I have added the contact section from the template.